open-machine-learning-jupyter-book/deep-learning/rnn.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>No picture is showed here? Something wrong?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>